### PR TITLE
Fix JaCoCo 0.8.3 on Java 9 and above

### PR DESCRIPTION
**Pull request by tobiasstadler**
#### Short description of what this resolves:
Starting from version 0.8.3 JaCoCo Java agent implementation uses the InjectedClassRuntime to define new class (named java.lang.$JaCoCo) in bootstrap class loader when running on JRE 9 and higher.

#### Changes proposed in this pull request:
Added a new case to JaCoCoConfiguration#isJacocoAgentActive

### CODE REVIEW

File: src/main/java/org/jboss/arquillian/extension/jacoco/client/configuration/JaCoCoConfiguration.java [hunk 1]
```diff
@@ -94,4 +94,13 @@ private static boolean hasClassJacocoAccessField(Class<?> clazz) {
         }
         return true;
     }
+
+    private static boolean hasClass(String className) {
+        try {
+            Class.forName(className);
+        } catch (Exception e) {
```
Comment (bartoszmajsak) [inline src/main/java/org/jboss/arquillian/extension/jacoco/client/configuration/JaCoCoConfiguration.java]: How about narrowing it to `ClassNotFoundException` and fail with other exceptions?

**Comment by bartoszmajsak** [comment]
Plus we might want to enhance travis build beyond `openjdk8` for that to be covered :)

**Comment by tobiasstadler** [comment]
Than you for your feedback. I will incorporate it in the next couple of days.

**Comment by tobiasstadler** [comment]
@bartoszmajsak Could you please have a look?

**Comment by bartoszmajsak** [comment]
Thanks, you might want to reset your master after this gets in as your PR is based on your fork's `master`

**Comment by bartoszmajsak** [comment]
@tobiasstadler I just released v1.0.0. It should land in Maven Central within the next 24h.

Thanks again for your contribution!

**Pull request merged**